tools/privacyidea-token-janitor

```diff
@@ -601,6 +616,28 @@ def find(last_auth, assigned, active, tokeninfo_key, tokeninfo_value,
         del tlist
 
 
+@manager.option('--yaml', dest='yaml',
+                help='Specify the YAML file with the previously exported tokens.')
+def updatetokens(yaml):
+    """
+    This can update existing tokens in the privacyIDEA system. You can specify a yaml file with the tokendata.
+    Can be used to reencrypt data, when changing the encryption key.
+    """
+    print("Loading YAML data. This may take a while.")
+    token_list = yaml_safe_load(open(yaml, 'r').read())
+    for tok in token_list:
+        del(tok["owner"])
```

Comment:
Should we also check if the assigned user matches? or is this not necessary here?

Reply:
No, this should probably not change.